mutating-admission-controller/util/util.go

```diff
@@ -0,0 +1,159 @@
+package util
```

Comment:
I see no reason to add a new package named `util`. I suggest we find a better name for the package (and the `.go` file).